java/kotlin-extractor/build.py

```diff
@@ -233,12 +235,13 @@ def compile_standalone(version):
             'build_standalone_' + version,
             version)
 
+compile_single_version = compile_embeddable if args.single_version_embeddable == True else compile_standalone
 
 if args.single_version:
-    compile_standalone(args.single_version)
+    compile_single_version(args.single_version)
 elif args.many:
     for version in kotlin_plugin_versions.many_versions:
         compile_standalone(version)
         compile_embeddable(version)
 else:
-    compile_standalone(kotlin_plugin_versions.get_single_version())
+    compile_single_version(kotlin_plugin_versions.get_single_version())
```

Comment:
I'm not sure it makes sense to support `--single-version-embeddable` without `--single-version`, in which case `compile_single_version` could be inlined into the `args.single_version` case.

Reply:
Done